docs/ops/state/savepoints.md

```diff
@@ -110,14 +110,11 @@ For example with a `FsStateBackend` or `RocksDBStateBackend`:
 /savepoints/savepoint-:shortjobid-:savepointid/...
 {% endhighlight %}
 
+After <a href="https://issues.apache.org/jira/browse/FLINK-5763">FLINK-5763</a> savepoint is self-contained and relocatable now, you can move the file and restore from any location.
```

Comment:
```suggestion
Since Flink 1.11.0, savepoints are self-contained and relocatable (see <a href="https://issues.apache.org/jira/browse/FLINK-5763">FLINK-5763</a> for details). You can move the file and restore from any location.
```